vllm/worker/model_runner.py

```diff
@@ -90,6 +90,12 @@ class ModelInputForGPU(ModelRunnerInputBase):
     request_ids_to_seq_ids: Optional[Dict[str, List[int]]] = None
     finished_requests_ids: Optional[List[str]] = None
     virtual_engine: int = 0
+    
+    slot_mapping: Optional[torch.Tensor] = None
```

Comment:
I can not find reference here. Seems not directly related to vineyard changes?

Reply:
seq_group_metadata_list should be the only field being added. The rest of the fields are ported from older versioned VLLM and I'll remove those.